### ISSUE TITLE
Allow warp on AUR

### ISSUE THREAD

**Issue by owofied** [warpdotdev/Warp#5014]
### Dupe Check

- [X] I have [searched Warp feature requests](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3A%22Feature+requests%22) and there are no duplicates

### Describe the solution you'd like?

The title speaks for itself

I would like to propose making warp available in the AUR. While I'm aware that there is an official pacman package, but it requires installing it by downloading the `.pkg.tar.zst` file from the warp.dev website and installing it using `sudo pacman -U` as discussed in #4495, I personally think that this is not an ideal solution for most people as updates are not installed along with system upgrades. Pretty much the same goes for the official pacman repositories as mentioned in the warp [documentation](https://docs.warp.dev/getting-started/getting-started-with-warp), not everyone is comfortable with adding repositories that are not officially provided by their distribution, especially when we have AUR which is an **officially** provided repository with user maintained packages, which makes me think that an AUR package for warp would be more than just an ideal solution for Arch Linux users, as making us add a 3rd party repository or installing it using `.pkg.tar.zst` file completely misses the point of AUR and the distribution itself.

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by dannyneira** [comment]
Hi @owofied thanks for submitting this feature request! 

To anyone else interested in this feature, please add a 👍 to the original post at the top to signal that you want this feature, and subscribe if you'd like to be notified.

**Comment by xPMo** [comment]
If you check [their Install page](https://docs.warp.dev/getting-started/getting-started-with-warp) and click on the Linux tab, you can see that they serve a pacman repo which you can get updates from, so `pacman -Syu` is all you need. Copying the instructions:

```bash
sudo sh -c "echo -e '\n[warpdotdev]\nServer = https://releases.warp.dev/linux/pacman/\$repo/\$arch' >> /etc/pacman.conf"
sudo pacman-key -r "linux-maintainers@warp.dev"
sudo pacman-key --lsign-key "linux-maintainers@warp.dev"
sudo pacman -Sy warp-terminal
```

**Comment by owofied** [comment]
@xPMo I also addressed that in my feature request

**Comment by cg505** [comment]
Aside: please do not recommend `pacman -Sy` in the installation instructions, as it can lead to partial system updates. (For example, it could trigger a glibc update without updating all the other packages that depend on glibc.)

**Comment by banekondic1996** [comment]
@cg505 I agree. I'm one that doesn't like updating system for longer periods. And when someone puts -Sy, it always annoys me
